server/views/components/interview/index.njk

```diff
@@ -0,0 +1,12 @@
+<div class="interview">
+  {% for qAndA in interview %}
+    <p class="interview__question">
+      <span class="interview__participant">{{ qAndA.question.participant }}</span>
```

Comment:
Wondering if the whole thing should be a definition list. Spec says:
"The dl element represents an association list of ... name-value groups ... Name-value groups may be ... questions and answers"